Makefile

```
@@ -384,6 +394,10 @@ $(OCRD_PAGETOPDF): ocrd_pagetopdf $(BIN)/ocrd
 endif
 
 ifneq ($(findstring ocrd_calamari, $(OCRD_MODULES)),)
+install-models: install-models-calamari
+.PHONY: install-models-calamari
+install-models-calamari: $(BIN)/ocrd
+	. $(ACTIVATE_VENV) && ocrd resmgr download ocrd-calamari-binarize '*'
```

Shouldn't that be…
```suggestion
	. $(ACTIVATE_VENV) && ocrd resmgr download ocrd-calamari-recognize '*'
```
…actually?